CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_greenhouse_source v0.8.0
+[PR #31](https://github.com/fivetran/dbt_greenhouse_source/pull/31) includes the following updates:
+
+## Breaking Changes
+- Casted `*_id` fields as strings to ensure compatibility in joins downstream.
```

Comment:
When reviewing the `*_id` fields original datatypes it seems the majority are integers ([example](https://github.com/fivetran/dbt_greenhouse_source/blob/ab89d72f57ec4be3f51299fbd3f89bc9e5f01f40/macros/get_scheduled_interviewer_columns.sql#L5-L7)). I'm on board with switching to string for more flexibility. However, I feel we should call this out in the CHANGELOG. Can you add a sub bullet here that says the majority of these fields were integers previously.